### ISSUE TITLE
UndefVarError: async_steal_repl_backend not defined

### ISSUE THREAD

**Issue by xximranxx** [timholy/Revise.jl#118]
Hi

I just downloaded julia 0.7 (v"0.7.0-beta2.0") and added Revise to the .julia/config/startup.jl as described here: https://timholy.github.io/Revise.jl/latest/config.html

However, I get the error that Revise.async_steal_repl_backend is not defined

If I instead do:
`@async Revise.steal_repl_backend()`
Revise just does not work

At times, I was getting an error that Revise could not be loaded during startup of julia (as a warning), but that warning has disappeared, though I do not know why

**Comment by timholy** [comment]
Yes, master was updated but I hadn't tagged. Does it work if you check out `master`?

Alternatively just wait until https://github.com/JuliaLang/METADATA.jl/pull/16263 merges and do a `Pkg.update`.

**Comment by timholy** [comment]
(BTW that has also fixed the broken link on the stable docs; I had never released a tagged version that used Documenter.)
